eclass/scons-utils.eclass

```diff
@@ -88,24 +81,10 @@
 # Much like EXTRA_EMAKE, this is not supposed to be used in make.conf
 # and not in ebuilds!
 
-# @ECLASS_VARIABLE: USE_SCONS_TRUE
-# @DESCRIPTION:
-# DEPRECATED: use usex instead
-#
-# The default value for truth in scons-use() (1 by default).
-: ${USE_SCONS_TRUE:=1}
-
-# @ECLASS_VARIABLE: USE_SCONS_FALSE
-# @DESCRIPTION:
-# DEPRECATED: use usex instead
-#
-# The default value for false in scons-use() (0 by default).
-: ${USE_SCONS_FALSE:=0}
-
 # -- EAPI support check --
 
 case ${EAPI:-0} in
-	0|1|2|3|4|5|6|7) ;;
+	7|8) ;;
 	*) die "EAPI ${EAPI} unsupported."
 esac
 
```

Comment:
wrt SCONS_MIN_VERSION: let's just default to 4.4.0 (current stable) but allow override.

That is, we can unconditionally do:
: ${WANT_AUTOMAKE:=4.4.0}
SCONS_DEPEND=">=dev-util/scons-${SCONS_MIN_VERSION}"

It's a lot simpler.

Reply:
Man, that's a major jump in automake releases xP.